ui/src/buckets/components/UpdateBucketOverlay.tsx

```diff
@@ -61,7 +61,7 @@ const UpdateBucketOverlay: FunctionComponent<Props> = ({
         handleClose()
         return
       }
-      setBucketDraft(resp.data)
+      setBucketDraft(resp.data as Bucket)
```

Comment:
setting this type here as a way of resolving a TS error that stemmed from a disparity between the expectations of the `useState` func and the data type being returned by the `getBucket` function